requirements/default.txt

```diff
@@ -1,6 +1,6 @@
 numpy>=1.16.5
 scipy>=1.2.3
-matplotlib>=3.0.3
+matplotlib>=3.0.1
```

Comment:
Is this change intentional?

Reply:
Woops, no, thanks for catching that; fixed.